pyspecdata/fourier/convolve.py

```diff
@@ -38,7 +48,7 @@ def convolve(self,axisname,filterwidth,convfunc = (lambda x,y: exp(-(x**2)/(2.0*
         print("detect self in time domain, already FT'd")
         self.ft(axisname)
     x = self.fromaxis(axisname)
-    myfilter = convfunc(x,1/filterwidth)
+    myfilter = convfunc(x,filterwidth)
```

Comment:
I would roll this back -- I actually changed it to `1/filterwidth` two commits back, as explained elsewhere.  If you don't do this, it breaks the old behavior of the convolve function.

Reply:
we are having a discussion on Slack, but for now I will keep this as `1/filterwidth`

Reply:
yes, I have an update below